packages/@lwc/synthetic-shadow/src/polyfills/document-shadow/polyfill.ts

```diff
@@ -43,6 +44,12 @@ function elemFromPoint(this: Document, left: number, top: number) {
 // https://github.com/Microsoft/TypeScript/issues/14139
 Document.prototype.elementFromPoint = elemFromPoint as (left: number, top: number) => Element;
```

Comment:
Same as above, I would prefer to avoid touching `elemFromPoint` in this PR. Also the issue is fixed, yes? I noticed that I can remove the `as ...` part and it works fine.

Reply:
The issue is fixed indeed. Let fix this in a follow-up PR.